### PR TITLE
Bug 1341954 support limited to x86_64 servers

### PR DESCRIPTION
**Pull request by bfallonf**
As per https://bugzilla.redhat.com/show_bug.cgi?id=1341954

From an email conversation and the BZ info, I don't think anything else needs to be added.

@tpoitras @ahardin-rh @tnguyen-rh Any issues?

### CODE REVIEW

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/install/prerequisites.adoc[Installing -> Prerequisites]
|Added Important box to the xref:../install_config/install/prerequisites.adoc#system-requirements[System Requirements] section.
%

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -106,6 +106,11 @@ back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
 |===
 
+[IMPORTANT]
+====
+OpenShift supports only x86_64 servers.
+====
```
Comment (tpoitras) [inline install_config/install/prerequisites.adoc]: Use product-title macro. Also, I recommend expansion to quantify what x86_64 is with "architecture":

{product-title} only supports servers with x86_64 architecture.

**Comment by tpoitras** [comment]
@bfallonf - left a comment

**Comment by bfallonf** [comment]
@tpoitras You're right, that's much better. And I really need to remember to do the product-title thing...

Cool. Thanks. I'll merge.

**Pull request merged**